lib/generators/rails_template18f/terraform/templates/terraform/main.tf.tt

```diff
@@ -0,0 +1,94 @@
+locals {
+  cf_org_name     = "<%= cloud_gov_organization %>"
+  app_name        = "<%= app_name %>"
+  space_deployers = setunion([var.cf_user], var.space_deployers)
+}
+
+module "app_space" {
+  source = "github.com/gsa-tts/terraform-cloudgov//cg_space?ref=v2.0.0"
+
+  cf_org_name   = local.cf_org_name
+  cf_space_name = var.cf_space_name
+  allow_ssh     = var.allow_space_ssh
+  deployers     = local.space_deployers
+  developers    = var.space_developers
+}
+# temporary method for setting egress rules until terraform provider supports it and cg_space module is updated
+data "external" "set-app-space-egress" {
+  program     = ["/bin/sh", "set_space_egress.sh", "-t", "-s", module.app_space.space_name, "-o", local.cf_org_name]
+  working_dir = path.module
+  depends_on  = [module.app_space]
+}
+
+module "database" {
+  source = "github.com/gsa-tts/terraform-cloudgov//database?ref=v2.0.0"
+
+  cf_space_id   = module.app_space.space_id
+  name          = "${local.app_name}-rds-${var.env}"
+  rds_plan_name = var.rds_plan_name
+  # depends_on line is needed only for initial creation and destruction. It should be commented out for updates to prevent unwanted cascading effects
```

Comment:
[question (nb)] (because I might forget to check) wondering if there's something about this in readme, if there should be if not. Also should the same note be on L20?

Reply:
I don't think there is anything in the README. I figured in the code was better because if you're not messing with the code then you won't need to change anything. I'm actually finding that in most cases you don't need to comment it out, so I think the change that should be made is ~`It should be commented out...`~ -> `It can be commented out if the plan has unwanted cascading effects`

Reply:
Makes sense, thanks.